libfreerdp/core/tpkt.c

```diff
@@ -132,8 +132,14 @@ BOOL tpkt_read_header(wStream* s, UINT16* length)
 	return TRUE;
 }
 
-BOOL tpkt_ensure_stream_consumed_(wStream* s, UINT16 length, const char* fkt)
+BOOL tpkt_ensure_stream_consumed_(wStream* s, size_t length, const char* fkt)
 {
+	if (length > UINT16_MAX)
+	{
+		WLog_ERR(TAG, "[%s] length %u" PRIuz " > %" PRIu16, length, UINT16_MAX);
```

Comment:
This will crash - first print specifier is a `%s` but the first parameter is `length` (`size_t`)

Reply:
fixed.